packages/fluent-editor/src/modules/ai/icons.ts

```
@@ -0,0 +1,350 @@
+// 输入框思考时左侧图标
+export const THINK_ICON = `<svg viewBox="0 0 22 18.207" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="22" height="18.2070312" fill="none" customFrame="#000000">
+	<g id="编组 17">
```

ESLint

[error] Unexpected tab character.

(style/no-tabs)